components/style/values/mod.rs

```diff
@@ -60,6 +60,23 @@ macro_rules! no_viewport_percentage {
     };
 }
 
+/// A macro for implementing `ComputedValueAsSpecified`, `Parse`
+/// and `HasViewportPercentage` traits for the enums defined
+/// using `define_css_keyword_enum` macro.
```

Comment:
Please reference that a proper solution would be to move `Parse` to style_traits, but that may not be trivial.